server/jwt_test.go

```diff
@@ -2907,30 +2900,25 @@ func TestAccountNATSResolverFetch(t *testing.T) {
 	require_1Connection := func(port int, creds string) {
 		t.Helper()
 		url := fmt.Sprintf("nats://localhost:%d", port)
```

Comment:
Recommend that you use `127.0.0.1` in cases where possible (like no TLS that would require hostname to be specified as localhost because that is what is in cert) because it can speed up tests on platforms where localhost would resolve to both ipv6/v4 because the test servers usually listen on `127.0.0.1`. This comment is true for all places where you have constructed an url with `localhost`.

Even better: instead of accepting `port int`, take `url string` here and have the caller pass `s.ClientURL()` (you can save it if used multiple times, like `saUrl := sA.ClientURL()` and then `require_1Connection(saUrl, ..)` or `connect(saUrl, ...)`, etc..

Reply:
fixed